reference/5.1/Microsoft.PowerShell.Core/About/about_Language_Keywords.md

```diff
@@ -251,42 +249,44 @@ exit
 exit <exitcode>
 ```
 
-When you use `powershell.exe` with the **File** parameter, the .ps1 (script)
+When you use `powershell.exe` with the **File** parameter, the `.ps1` (script)
 file itself should include instructions for handling any errors or exceptions
-that occur while the script is running. You should only use the exit statement
+that occur while the script is running. You should only use the `exit` statement
 to indicate the post-execution status of the script.
 
-In PowerShell, the exit statement sets the value of the `$LASTEXITCODE`
+Any number between `[int]::MinValue` and `[int]::MaxValue` is allowed.
+
+In PowerShell, the `exit` statement sets the value of the `$LASTEXITCODE`
 variable. In the Windows Command Shell (cmd.exe), the exit statement sets the
 value of the `%ERRORLEVEL%` environment variable.
 
+Any argument that is non-numeric or outside the platform-specific range is
+translated to the value of `0`.
+
 In the following example, the user sets the error level variable value to 4 by
-adding `exit 4` to the script file _test.ps1_.
+adding `exit 4` to the script file `test.ps1`.
 
 ```cmd
 C:\scripts\test>type test.ps1
 1
-
 2
-
 3
-
 exit 4
 
 C:\scripts\test>powershell -file ./test.ps1
 1
-
 2
-
 3
 
 C:\scripts\test>echo %ERRORLEVEL%
 4
 ```
 
-When you run `powershell.exe -File <path to a script>`, the exit statement
-sets the `%ERRORLEVEL%` variable to a value other than zero. If you have an
-unhandled exception in your script, `%ERRORLEVEL%` is set to the value of 1.
+When you run `pwsh.exe -File <path to a script>` and the script file terminates
+with an `exit` command, the exit code is set to the numeric argument used with
+the `exit` command. If the script has not `exit` statement, the exit code is
+always `0`, when the script completes without error, or `1` when the script
```

Comment:
I don't think these commas are needed. I think it fragments the sentence too much.

Reply:
Fixed